html/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Walter Velasquez</title>
+    <link rel="stylesheet" href="../css/style.css" />
+    <script src="https://kit.fontawesome.com/62ea397d3a.js" crossorigin="anonymous"></script>
+  </head>
+ 
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="nav_mobile">
+          <div class="logo">Sir V</div>
+          <div class="nav_mobile-btn 
+          fas fa-bars"></div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="headline">
+      <div class="presentation">
+        <h1>Walter Velasquez</h1>
+        <h2>Software developer</h2>
+      </div>
+      
+      <div class="sup_text">
+        <p class="st">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        <img alt="divider vector" src="../img/Vector.svg" class="divider_a">
+
+      <div class="sup_skills">
+        <ul class="sup_list">
+          <li class="list_title">Language</li>
+          <li class="sup_li">Javascript</li>
+          <li class="sup_li">Ruby</li>
+          <li class="sup_li">Html</li>
+          <li class="sup_li">Css</li>
+        </ul>
+        <ul class="sup_list">
+          <li class="list_title">Frameworks</li>
+          <li class="sup_li">Bootstrap</li>
+          <li class="sup_li">Ruby on Rails</li>
+          <li class="sup_li">RSPec</li>
+          <li class="sup_li">CapyBara</li>
+          <li class="sup_li">Selenium</li>
+        </ul>
+        <ul class="sup_list">
+          <li class="list_title">Skills</li>
+          <li class="sup_li">Codekit</li>
+          <li class="sup_li">Github</li>
+          <li class="sup_li">Codepen</li>
+          <li class="sup_li">Gitlab</li>
+          <li class="sup_li">Terminal</li>
+        </ul>
+      </div>
+      <img alt="divider vector" src="../img/Vector.svg" class="divider_b">
+      </div>
+
+      <div class="sup_items">
+        <div class="social_media">
+          <div class="social_icon"><a href="#"><img alt="My Github" src="../img/icons/icongit.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My in icon" src="../img/icons/iconin.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My angelist" src="../img/icons/iconangel.svg"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My twitter" src="../img/icons/icontw.png"></a></div>
+          <div class="social_icon"><a href="#"><img alt="My Medium" src="../img/icons/iconm.png"></a></div>
+        </div>
```

Comment:
- [Required] :
Please make sure that the social media icons ("GitHub", "Twitter") are in a `ul` tag, each of them in a `li `tag. ; each icon has a link (`a` tag)